src/main/resources/templates/db_scripts/schema.json

```diff
@@ -33,10 +33,6 @@
           "fieldName" : "username",
           "tOps" : "ADD"
         },
-        {
-          "fieldName" : "id",
-          "tOps" : "ADD"
-        },
```

Comment:
Change `"tOps" : "ADD"` to `"tOps" : "DELETE"` to delete the indexes on module upgrade.